packages/codegen-ui-react/lib/utils/forms/validation.ts

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable */
+import {
+  FieldValidationConfiguration,
+  ValidationResponse,
+  ValidationTypes,
+} from '@aws-amplify/codegen-ui/lib/types/form/form-validation';
+
+export const validateField = (value: any, validations: FieldValidationConfiguration[]): ValidationResponse => {
```

Comment:
Since validation function is static, we shouldnt need to dynamically generate the function with ts. We can just read/write file during codegen.